app/src/main/java/com/github/sdp/mediato/data/Database.java

```diff
@@ -214,4 +221,45 @@ public static CompletableFuture<String> addCollection(String username, Collectio
         return future;
     }
 
+    /**
+     * Method to follow a user
+     *
+     * @param myUsername       the current user's username
+     * @param usernameToFollow the username of the user to follow
+     */
+    public static void followUser(String myUsername, String usernameToFollow) {
+        CompletableFuture<String> future = new CompletableFuture<>();
+        database.getReference()
+                .child(USER_PATH + myUsername + FOLLOWING_PATH + usernameToFollow).setValue(true)
+                .addOnCompleteListener(task -> {
+                    System.out.println(myUsername + " is now following " + usernameToFollow);

```

Comment:
consider converting the statement lambda to expression lambda (same remark for all  addOnCompleteListener in this PR)

Reply:
Done